OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/http/impl/HttpClient.kt

```diff
@@ -46,29 +46,39 @@ internal class HttpClient(
         url: String,
         body: JSONObject,
         headers: OptionalHeaders?,
-    ): HttpResponse = makeRequest(url, "POST", body, _configModelStore.model.httpTimeout, headers)
+    ): HttpResponse {
+        return makeRequest(url, "POST", body, _configModelStore.model.httpTimeout, headers)
+    }
```

Comment:
I ran into some issues when the change actually has a difference but I couldn't reproduce it anymore. It could be an implementation issue. I changed these lines back agian.